app/src/main/java/bubbleheads/buy_cook/CategoryList.java

```diff
@@ -0,0 +1,38 @@
+package bubbleheads.buy_cook;
+
+import java.util.ArrayList;
+
+public class CategoryList {
+
+    private static final ArrayList<Category> CATEGORY_LIST = new ArrayList<Category>() {
+        {
+            add(new Category("Soups")); //1
+            add(new Category("Pizza")); //2
```

Comment:
actually I would pass id to constructor. I mean when you'll be getting your data from server they should have id already inside. Ask me again in chat if not clear